dictionary/mix.exs

```diff
@@ -14,6 +14,7 @@ defmodule Dictionary.Mixfile do
   def application do
     [
       extra_applications: [:logger]
+      mod: {Dictionary, []}
```

Comment:
# -30

The application cannot run because there's a comma missing on line 16, and because of the typo and syntax error in dictionary.ex, the undefined `opts` in supervisor.ex, the syntax error on line 2 of agent.ex, and ??? (at this point I stopped fixing stuff)